app/apptesting/test_helpers.go

```diff
@@ -356,6 +357,14 @@ func GenerateTestAddrs() (string, string) {
 	return validAddr, invalidAddr
 }
 
+// Grabs an admin address to test validate basic on admin txs
+func GetAdminAddress() (address string, ok bool) {
+	for address := range utils.Admins {
+		return address, true
```

Comment:
nit: is this just going to always return the first admin address? kinda confusing to do that in a loop; possible it's a golang idiom

Reply:
the admins list is a map haha (not sure why tbh) but this is the only way to grab the "first item" from a map as far as I know

Reply:
actually I guess is does make sense as a map cause we're generally starting from a supplied address and checking if it's an admin